app/ChordInspector/src/main/java/org/jjazz/chordinspector/ChordListener.java

```diff
@@ -51,20 +55,43 @@ public ChordListener(ChordInspectorPanel editor)
         cap = CL_ContextActionSupport.getInstance(Utilities.actionsGlobalContext());
         cap.addWeakSelectionListener(this);
         selectionChange(cap.getSelection());
+
+        // Listen to transposition option changes
+        playbackSettings = PlaybackSettings.getInstance();
+        playbackSettings.addPropertyChangeListener(PlaybackSettings.PROP_CHORD_SYMBOLS_DISPLAY_TRANSPOSITION, this);
+        setTransposition(playbackSettings.getChordSymbolsDisplayTransposition());
     }
 
     public void cleanup()
     {
         cap.removeWeakSelectionListener(this);
+
+        playbackSettings.removePropertyChangeListener(PlaybackSettings.PROP_CHORD_SYMBOLS_DISPLAY_TRANSPOSITION, this);
         if (chordSymbol != null)
         {
             chordSymbol.removePropertyChangeListener(this);
         }
     }
 
-    // -----------------------------------------------------------------------------
-    // CL_ContextActionListener interface
-    // -----------------------------------------------------------------------------   
+    private void setTransposition(int newTransposition)
+    {
+        transposition = newTransposition;
+        if (chordSymbol != null)
+        {
+            useTransposedChord(chordSymbol.getData(), transposition);
+        }
+    }
+
+    private void useTransposedChord(ExtChordSymbol currentChord, int currentTransposition)
```

Comment:
I don't see the point of using method parameters, each time you call this method you pass global variables.  Method could be just called `updateEditorModel()`